generated/src/aws-cpp-sdk-dynamodb/include/aws/dynamodb/DynamoDBClient.h

```diff
@@ -42,13 +43,42 @@ namespace DynamoDB
       Aws::Crt::Variant<smithy::SigV4AuthScheme>,
       DynamoDBEndpointProviderBase,
       smithy::client::JsonOutcomeSerializer,
-      smithy::client::JsonOutcome>,
+      smithy::client::JsonOutcome,
+      Aws::Client::DynamoDBErrorMarshaller>,
     Aws::Client::ClientWithAsyncTemplateMethods<DynamoDBClient>
   {
     public:
       static const char* GetServiceName();
       static const char* GetAllocationTag();
       inline const char* GetServiceClientName() const override { return "DynamoDB"; }
+      
+      DynamoDBClient(const DynamoDBClient& other);
+
+      DynamoDBClient(DynamoDBClient&& other);
+
+      DynamoDBClient& operator=(const DynamoDBClient& other)
+      {
+        if(this == &other) 
+        {
+            return *this;
+        }
+        AwsSmithyClientT::operator=(other);
+        Aws::Client::ClientWithAsyncTemplateMethods<DynamoDBClient>::operator=(other);
+        init(m_clientConfiguration);
+        return *this;
+      }
+
+      DynamoDBClient& operator=(DynamoDBClient&& other)
+      {
+        if(this == &other) 
+        {
+            return *this;
+        }
+        AwsSmithyClientT::operator=(std::move(other));
+        Aws::Client::ClientWithAsyncTemplateMethods<DynamoDBClient>::operator=(std::move(other));
```

Comment:
dont think you can call std::move on a r value twice [without entering UB land](https://godbolt.org/z/x7rWsvnvr).